messenger.java

```diff
@@ -32,8 +41,8 @@ public void initialise()
 			panel.add(ipui);
 			okbtn.setBounds(20,70,200,40);
 			panel.add(okbtn);
```

Comment:
i tried to add panel.add(message) here, so that it may show up in the initial UI, but for some reason it did not show up, so removed that statement altogether